pyproject.toml

```diff
@@ -1,6 +1,80 @@
 [build-system]
-requires = ["setuptools >= 46.4.0", "wheel", "fastentrypoints"]
-build-backend = "setuptools.build_meta"
+requires = ["flit_core >=3.4,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.module]
+name = "aiida_testing"
+
+[project]
+name = "aiida-testing"
+dynamic = ["version", "description"]
+authors = [
+    {name = "Dominik Gresch"},
+    {name = "Leopold Talirz"},
+    {name = "Jens Bröder"},
+    {name = "Philipp Rüßmann"},
+    {name = "AiiDA team"},
+]
+license = {file = "LICENSE.txt"}
+readme = "README.md"
+classifiers = [
+    "Development Status :: 3 - Alpha",
+    "Environment :: Plugins",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: MIT License",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Topic :: Scientific/Engineering :: Physics",
+    "Framework :: AiiDA",
+    "Framework :: Pytest",
+]
+keywords = [
+    "AiiDA",
+    "testing",
+    "pytest",
+    "mock",
+    "cache",
+]
+urls = {Homepage = "https://aiida-testing.readthedocs.io/"}
+requires-python = ">=3.7"
+dependencies = [
+    "aiida-core>=1.0.0,<3.0.0",
+    "pytest>=3.6",
+    "voluptuous~=0.12",
+]
+
+[project.optional-dependencies]
+docs = ["sphinx", "sphinx-rtd-theme"]
+testing = [
+    "pgtest~=1.3.1",
+    "aiida-diff",
+    "pytest-datadir",
+]
+pre_commit = [
+    "pre-commit",
+    "pylint~=2.12.2",
+    "mypy==0.930",
+    "types-PyYAML",
+]
+dev = [
+    "sphinx",
+    "sphinx-rtd-theme",
+    "pgtest~=1.3.1",
+    "aiida-diff",
+    "pytest-datadir",
+    "pre-commit",
+    "pylint~=2.12.2",
+    "mypy==0.930",
+]
+
+[project.scripts]
+aiida-mock-code = "aiida_testing.mock_code._cli:run"
+
+[project.entry-points."pytest11"]
+aiida_mock_code = "aiida_testing.mock_code"
+aiida_export_cache = "aiida_testing.export_cache"
```

Comment:
FYI you don't need to load via `pytest_plugins`, if you include this entry point 😉

Reply:
Oh, nice feature!

Also a bit "dangerous" - I guess any package can now drop any fixture into your pytest setup ;-)

Reply:
well you likely wouldn't be installing this package if you didn't want to use the fixtures 😄